src/gnssro/gnssro_bufr2ioda.py

```diff
@@ -227,12 +233,15 @@ def get_obs_data(bufr, profile_meta_data, add_qc, addLSW, record_number=None):
     obs_data[('bendingAngle', "PreQC")] = np.full(krepfac[0], 0, dtype=ioda_int_type)
 
     # (geometric) height is read as integer but expected as float in output
-    height = codes_get_array(bufr, 'height', ktype=float)
-
-    # get the refractivity
-    refrac = codes_get_array(bufr, 'atmosphericRefractivity')[0::2]
-    refrac_err = codes_get_array(bufr, 'atmosphericRefractivity')[1::2]
-    refrac_conf = codes_get_array(bufr, 'percentConfidence')[sum(krepfac[:1])+1:sum(krepfac[:2])+1]
+    if only_bang:
```

Comment:
The refractivity is overwritten with the bending angle. This could be confusing later on. Could we fill all the `refrac` array with missing values instead?

Reply:
good idea,,,, or exclude it entirely is the other option and my preference,, do not know if that works 

missing may be a initial step